src/Analysis.R

```diff
@@ -60,25 +51,33 @@ penguins_data <- penguins_data |> drop_na()
 
 ## ----descriptives----
 
-# NOTE: Descriptives not computed, as `gtsummary` package is too heavy
+## Simple summary:
+penguins_data |> summary()
 
-## Create descriptive statistics table:
-# descriptive_table <- penguins_data |> tbl_summary()
-# descriptive_table
-## TODO: Uncomment if using {gtsummary}
+## Create a contingency table:
+counts_table <-
+  penguins_data |>
+  group_by(species, island, sex) |>
+  count() |>
+  ungroup()
 
-## Create contingency table of sex and species:
-# contingency_table <- penguins_data |> tbl_cross(row = sex, col = species)
-# contingency_table
-## TODO: Uncomment if using {gtsummary}
+## Create descriptive statistics table:
+summary_metrics <-
+  penguins_data |>
+  group_by(species, island, sex) |>
+  summarise(mean_bill_length = mean(bill_length_mm),
+            se_bill_length = sd(bill_length_mm)/sqrt(n()),
+            mean_flipper_length = mean(flipper_length_mm),
+            se_flipper_length = sd(flipper_length_mm)/sqrt(n()),
+            mean_body_mass = mean(body_mass_g),
+            se_body_mass = sd(body_mass_g)/sqrt(n())
+  ) |>
+  ungroup()
 
 
 ## ----modeling----
 
 bodymass_fit <- lm(body_mass_g ~ sex * species, data = penguins_data)
-## TODO: Add covariates maybe?
+## TODO: Add covariates? Test model assumptions?
+
 
-# NOTE: Output table not created, as `gtsummary` package is too heavy
-# bodymass_coef_table <- bodymass_fit |> tbl_regression()
-# bodymass_coef_table
-## TODO: Uncomment if using {gtsummary}
```

Comment:
This is a proposal for a "change they may implement and commit", I'd rather leave it.